packages/valory/skills/agent_performance_summary_abci/handlers.py

```diff
@@ -39,12 +49,334 @@
 from packages.valory.skills.abstract_round_abci.handlers import (
     TendermintHandler as BaseTendermintHandler,
 )
+from packages.valory.skills.agent_performance_summary_abci.dialogues import (
+    HttpDialogue,
+)
+from packages.valory.skills.agent_performance_summary_abci.graph_tooling.queries import (
+    GET_TRADER_AGENT_DETAILS_QUERY,
+)
+from packages.valory.skills.staking_abci.rounds import SynchronizedData
+
+
+class HttpMethod(Enum):
+    """Http methods"""
+
+    GET = "get"
+    HEAD = "head"
+    POST = "post"
+
```

Comment:
Can be reused from the dependency skill instead:

https://github.com/valory-xyz/trader/blob/b5fcba4b379a5fb2ebb17604958abb8f2632d03a/packages/valory/skills/chatui_abci/handlers.py#L73-L78